NEWS

```diff
@@ -1497,7 +1497,7 @@ Jython 2.2 beta1
     - new-style classes and their instances are Java serializable
     - most new-style builtin types can be pickleds
     - __slots__ for new-style classes
-    - xml modules from pyxml readded
+    - xml modules from pyxml read
```

Comment:
I think the author meant "added back", but it was too long ago to make research worthwhile.